test-executor/src/main/java/com/cognifide/aet/executor/xmlparser/xml/utils/EscapeUtils.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Automated Exploratory Tests
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.aet.executor.xmlparser.xml.utils;
+
+import org.apache.commons.lang3.StringEscapeUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class EscapeUtils {
+
+  private static final Pattern URL_TAG_PATTERN = Pattern.compile("(<url)(\\s+)(.*)(\\s*)(/>)");
```

Comment:
This regex is used to replace URL's with escaped version before actual parsing occurs.